x-pack/plugins/secops/public/store/local/inputs/reducer.ts

```diff
@@ -25,7 +25,7 @@ export const initialInputsState: InputsState = {
     timerange: {
       kind: 'absolute',
       from: moment()
-        .subtract(1, 'hour')
+        .subtract(1, 'day')
```

Comment:
Take a look at this function:
https://github.com/elastic/kibana/blob/feature-secops/x-pack/plugins/secops/public/components/range_date_picker/quick_select_popover/commonly_used.tsx#L104

That's wired into the date time of 'Yesterday'. I think maybe we want that logic here rather than a rolling day?

Reply:
optional: a comment could make it clear that this subtracts 24 hours from "now", vs "it returns the start of the previous day":

```
 .subtract(1, 'day') // subtracts 24 hours from 'now`
```

Reply:
Updated to include comment. Also @FrankHassanabad, this behavior mirrors the Quick Select's `Last 1 day` UI as opposed to the `Yesterday` shortcut.